ethportal-api/src/types/execution/header.rs

```diff
@@ -58,6 +55,18 @@ pub struct Header {
     pub base_fee_per_gas: Option<U256>,
     /// Withdrawals root from execution payload. Introduced by EIP-4895.
     pub withdrawals_root: Option<H256>,
+    /// Blob gas used. Introduced by EIP-4844
+    pub blob_gas_used: Option<U64>,
+    /// Excess blob gas. Introduced by EIP-4844
+    pub excess_blob_gas: Option<U64>,
+    /// The hash of the parent beacon block's root is included in execution blocks, as proposed by
```

Comment:
Imo, just simply `The parent beacon block's root hash. Introduced by EIP-4788` is fine. If users want to understand more, they can go read the eip themselves